nixos/modules/config/xdg/portals/termfilechooser.nix

```diff
@@ -0,0 +1,79 @@
+{ config
+, pkgs
+, lib
+, ...
+}:
+let
+  cfg = config.xdg.portal.termfilechooser;
+  package = pkgs.xdg-desktop-portal-termfilechooser;
```

Comment:
I think a package option would fit better here, in case overrides are needed, or someone wants use a fork, etc

Reply:
If we end up keeping the module this is obviously something I'll add

Reply:
Add in the last commit